apps/frontend/src/styles/pageStyles.tsx

```diff
@@ -30,4 +30,29 @@ export const useStyles = () => ({
     color: colors.martletplaceBlack,
     "&:hover": { color: colors.martletplaceBlueHover },
   },
+
+  // Account page styles
+  drawer: {
+    width: 350,
```

Comment:
I haven't checked, but I'm assuming this css is not mobile friendly because it's hard coded at 350px. Also, using either 350 or 350px is okay, but it should be consistent